xs/src/libslic3r/MedialAxis.cpp

```diff
@@ -1265,7 +1255,7 @@ MedialAxis::ensure_not_overextrude(ThickPolylines& pp)
         //reduce width
         double reduce_by = boundsVolume / volume;
         for (ThickPolyline& polyline : pp) {
-            for (ThickLine l : polyline.thicklines()) {
+            for (ThickLine &l : polyline.thicklines()) {
```

Comment:
This still does nothing - thicklines() returns copy of polyline data.
(and if it would reference polyline internally, this code would reduce width of midpoints twice)

Reply:
should be okay now